src/components/molecules/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,59 @@
+import VenueChat from "components/molecules/VenueChat";
+import ChatsList from "components/molecules/ChatsList";
+import LiveSchedule from "components/molecules/LiveSchedule";
+import { useSelector } from "hooks/useSelector";
+import { useVenueId } from "hooks/useVenueId";
+import React, { useState } from "react";
+import { useFirestoreConnect } from "react-redux-firebase";
+import "./Sidebar.scss";
+
+const Sidebar = () => {
+  const venueId = useVenueId();
+  useFirestoreConnect({
+    collection: "users",
+    where: ["enteredVenueIds", "array-contains", venueId],
+    storeAs: "chatUsers",
+  });
+  const [tab, setTab] = useState(0);
+  const { privateChats, chatUsers } = useSelector((state) => ({
+    privateChats: state.firestore.ordered.privatechats,
+    chatUsers: state.firestore.data.chatUsers,
+  }));
+  const isEnabled = chatUsers && privateChats;
```

Comment:
Good usage of extracting the logic into a `const` + 'semanticly named variables'

[nit] A nit that I would tend to use here is to coerce it into a definite boolean (just to avoid any potential weird bugs somewhere else later), eg.

```suggestion
  const isEnabled = !!chatUsers && !!privateChats;
```

or

```suggestion
  const isEnabled = !!(chatUsers && privateChats);
```

Reply:
nit/stylistic/no agreed upon convention, so resolving this